### PR TITLE
updated to the latest version of dskit

### PR DESCRIPTION
**Pull request by vlad-diachenko**
**What this PR does / why we need it**:
Updated Loki with the latest version of dskit

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,7 @@
 * [7754](https://github.com/grafana/loki/pull/7754) **ashwanthgoli** index-shipper: add support for multiple stores.
 * [8662](https://github.com/grafana/loki/pull/8662) **liguozhong**: LogQL: Introduce `distinct`
 * [9813](https://github.com/grafana/loki/pull/9813) **jeschkies**: Enable Protobuf encoding via content negotiation between querier and query frontend.
+* [9920](https://github.com/grafana/loki/pull/9920) **vlad-diachenko**: Bump dskit version.
```
Comment (chaudum) [inline CHANGELOG.md:49]: I don't think we should make a changelog entry for updating a dependency, except is has some user-facing changes.

**Review by owen-d — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vlad-diachenko) [reply CHANGELOG.md:49]: I also was thinking about it )) it makes sense

File: docs/sources/configure/_index.md [hunk 2]
```diff
@@ -3624,7 +3624,19 @@ backoff_config:
   # CLI flag: -<prefix>.backoff-retries
   [max_retries: <int> | default = 10]
 
-# Enable TLS in the GRPC client. This flag needs to be enabled when any other
+# Initial stream window size. Values less than the default are not supported and
+# are ignored. Setting this to a value other than the default disables the BDP
+# estimator.
+# CLI flag: -<prefix>.initial-stream-window-size
+[initial_stream_window_size: <int> | default = 63KiB1023B]
```
Comment (chaudum) [inline docs/sources/configure/_index.md:3631]: :sob: 
 
```suggestion
[initial_stream_window_size: <int> | default = 64KiB]
```

Comment (vlad-diachenko) [reply docs/sources/configure/_index.md:3631]: // The default value of flow control window size in HTTP2 spec.
	defaultWindowSize = 65535

Comment (vlad-diachenko) [reply docs/sources/configure/_index.md:3631]: it looks like it's not a mistake

File: CHANGELOG.md (hunk 1 above)
Comment (vlad-diachenko) [reply CHANGELOG.md]: make format fixed it

File: docs/sources/configure/_index.md (hunk 2 above)
Comment (vlad-diachenko) [reply docs/sources/configure/_index.md]: make format

**Pull request merged**